VoteSystem.Console/ContextRegistration.cs

```diff
@@ -18,7 +27,13 @@ public bool SetPasswordInfo(string passportCode, int indefCode)
         {
             _passportCode = passportCode;
             _indefCode = indefCode;
+            _loggedInUser = _userRepository.GetUser(passportCode, indefCode);
             return true;
         }
+
+        public User GetLoggedUser()
```

Comment:
because you need logged in user all over the application. As soon as you have a context holding in formation about currently logged user - there is no need to re-fetch and re-fetch and re-fetch this user